packages/foreman/foreman-obsolete-packages/foreman-obsolete-packages.spec

```diff
@@ -0,0 +1,33 @@
+Name: foreman-obsolete-packages
+Version: 1.1
+Release: 0.1%{?dist}
+License: MIT
+Summary: A package to obsolete retired packages
+URL: https://github.com/theforeman/foreman-packaging
+BuildArch: noarch
+
+Obsoletes: rubygem-google-api-client < 0.33.2-3
+
+%description
+This package exists only to obsolete other packages which need to be removed
+from the distribution for some reason.
+
+%prep
+
+%build
+
+%install
+
+%files
+
+%changelog
+* Fri Jan 27 2023 Evgeni Golov - 1.1-0.1
```

Comment:
This kind of makes me wonder if we need to introduce foremandist in the release. Now the changelog goes back in version numbers, which isn't great. On the other hand, foremandist wouldn't solve that either.

Perhaps we should just drop the changelog and introduce it as version 0.1.

Reply:
I thought about 0.9, but yeah, anything below 1.0 would work.

Really, don't mind any specific version number, as long as we can smoothen the 3.3→3.4 upgrade.

Reply:
Let's go with 0.9 then.

Reply:
Like this?